### ISSUE TITLE
loss doesn't drop 

### ISSUE THREAD

**Issue by gzq17** [SuperStacie/JointVesselNet_VC-Net#2]
Hello, I downloaded your code, but the loss doesn't drop during training. I checked the input data and the corresponding label, there is no problem. And I also implemented it with pytorch, and I also encountered the same problem. Is there any training trick that I didn't notice?
